features/step_definitions/templates.js

```diff
@@ -5,7 +5,7 @@ const request = require('../support/request');
 const path = require('path');
 const Assert = require('chai').assert;
 
-module.exports = function () {
+module.exports = function server() {
```

Comment:
`module.exports = () => {` can not be used here. When used, I think that `this` will be undefined because `this` does not exist in this scope. [References is here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions)
Also, if omit the function name, then it will get a warning with eslint. It is no problem, but I think that it is better to deal with this. [References is here](http://eslint.org/docs/rules/func-names)
![2017-03-22 12 18 06](https://cloud.githubusercontent.com/assets/24538326/24180673/a6224420-0ef9-11e7-8d15-c48dff84672f.png)